### DISCUSSION TITLE
백신접종 예약내역 있을 경우

### DISCUSSION THREAD

**Discussion by 77mong** [kimytsc/covid-rest-vaccine-macro#64] [General]
안녕하세요

먼저 이렇게 좋은 프로그램 제작하여 제공해주셔서 감사드립니다.

바로 아래 글에서 답변해주신 내용 확인해보니, 백신 예약내역이 있는 경우 예약이 실패되는 경우가 발생한다고 말씀하셨는데,

현재 백신 사전예약 내역이 있을 경우, 이 프로그램으로는 잔여백신 예약이 힘든건지요??

3일째 계속 프로그램 돌려놓고 있는데 항상 실패하고 있어서 문의드립니다. 

감사합니다.

**Comment by kimytsc** [comment]
2021년 8월 16일 월요일부터 예방접종 사전예약자도 잔여백신 신청이 가능하며, 잔여백신을 신청할 경우 자동으로 사전예약이 취소된다고 발표되었습니다.
네이버와 카카오도 발빠르게 예약 가능하도록 조치되어 있을 것으로 생각되며, 해당 플랫폼의 API로만 잔여백신을 예약할 수 있기에 문제 없을 것으로 생각됩니다. 다른 종류의 매크로에서도 동일하게 동작하기에 예약이 가능합니다,

그리고 예약이 잘 안되더라도 꾸준히 시도해보시는걸 추천드립니다.
운이 좋아 10분만에 예약 성공하신 분도 계시지만 몇 일만에 성공한 케이스도 있으니 다른 메크로를 쓰시더라도 인내심을 가지고 도전해보시고, 성공하길 바라겠습니다!
